Add MOAR attributes

kostasmanionis opened the pull request:
This should resolve https://github.com/insin/babel-plugin-react-html-attrs/issues/1 + transform SVG attributes.

I kinda figured it would be nice if this also transformed SVG attributes. We're using this for building react SVG components.

hellatan commented on test/fixtures/basic/actual.js:
```diff
@@ -1,3 +1,6 @@
 <div id="test" class="test">
   <label for="test" class="test">Test</label>
+  <svg>
+    <path x="10" y="10" stroke-dasharray="5,10,5"/>
```
i would also test against a namespaced attribute like `xlink:role` or something

insin commented:
Thanks for the PR, looks good.

kostasmanionis commented:
I kinda lowballed on tests here, but I don't think there's much more to test here. 

Few things I'd like to point out:

We're gonna be transforming all the event handler attributes `<div onclick="doSomething();">Click me!</div>` but it won't work since react only accepts functions.

And we're gonna be transforming svg attributes on non svg elements. Not sure if this is a problem though.

insin commented:
On review, I'm not keen on having the event handlers in here, as that will never work in any HTML you're copying and pasting anyway.

I'm wondering if it would be cleaner to take the SVG props and spin them out into a new `babel-plugin-react-svg-attrs`.

kostasmanionis commented:
So I've removed every `onWhatever` attribute from the list.

Also removed svg attributes. I'm cool with spinning of svg to a separate pacakge, we could also make a preset for people who want to use both.

insin commented:
Nice, thanks.

Thinking the best option is probably to make this plugin configurable after dealing with #3.
